src/main/java/com/google/sps/servlets/SpoonacularNutritionServlet.java

```diff
@@ -32,15 +32,15 @@
 /** Servlet to take in dish name and return bulk recipe infomation */
 @WebServlet("/dishNutrition")
 public class SpoonacularNutritionServlet extends HttpServlet {
-  static String spoonacularPrefix = "https://api.spoonacular.com/recipes/guessNutrition";
-  static String spoonacularAPIKey = "cd2269d31cb94065ad1e73ce292374a5";
+  private static String spoonacularPrefix = "https://api.spoonacular.com/recipes/guessNutrition";
```

Comment:
Constant names should use "ALL_UPPER_CASE" format (https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)

Also we should use "private static final String"